### PR TITLE
[13.0] [MIG] sale_stock_sourcing_address 

### PR DESCRIPTION
**Pull request by nikul-serpentcs**

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@JordiBForgeFlow @HviorForgeFlow @AaronHForgeFlow Can you please take a look?

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_stock_sourcing_address/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Eficent
```
Comment (AaronHForgeFlow) [inline sale_stock_sourcing_address/readme/CONTRIBUTORS.rst]: ForgeFlow

File: sale_stock_sourcing_address/models/account_invoice_line.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
```
Comment (AaronHForgeFlow) [inline sale_stock_sourcing_address/models/account_invoice_line.py]: ForgeFlow S.L.

**Review by AaronHForgeFlow — APPROVED** [review]
Tests ok

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow @JordiBForgeFlow Ready to merge

**Review by HaraldPanten — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]
Reviewed

**Comment by AaronHForgeFlow** [comment]
Does this need retry or a fix?

**Comment by AaronHForgeFlow** [comment]
@dreispt could you retry?

File: sale_stock_sourcing_address/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Sale Stock Sourcing Address",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow S.L.," "Odoo Community Association (OCA)",
```
Comment (newtratip) [inline sale_stock_sourcing_address/__manifest__.py:7]:
```suggestion
    "author": "ForgeFlow S.L.,Odoo Community Association (OCA)",
```

**Review by newtratip — APPROVED** [review]

**Comment by rousseldenis** [comment]
@nikul-serpentcs Could you rebase ?

**Comment by AaronHForgeFlow** [comment]
it seems travis is not happy

**Comment by rousseldenis** [comment]
@nikul-serpentcs @JayVora-SerpentCS

**Comment by nikul-serpentcs** [comment]
@rousseldenis now the code is updated, Can you please take a look?

Test case failed based on  **sale_sourced_by_line** module.

**Comment by MiquelRForgeFlow** [comment]
@nikul-serpentcs please, edit https://github.com/OCA/sale-workflow/blob/13.0/.github/workflows/test.yml#L37 and add case specific for sale_sourced_by_line, and exclude it in the generic case.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -49,6 +49,13 @@ jobs:
           - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
             exclude: "sale_order_line_sequence"
             name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
+            exclude: "sale_sourced_by_line"
+            makepot: "true"
+            name: test with Odoo
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
+            exclude: "sale_sourced_by_line"
+            name: test with OCB
```
Comment (MiquelRForgeFlow) [inline .github/workflows/test.yml]:
```suggestion
            exclude: "sale_order_line_sequence,sale_sourced_by_line"
            name: test with OCB
          - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
            include: "sale_sourced_by_line"
            makepot: "true"
            name: test with Odoo
          - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
            include: "sale_sourced_by_line"
            name: test with OCB
```

Comment (MiquelRForgeFlow) [reply .github/workflows/test.yml]: sorry, add sale_sourced_by_line also in line 46

Comment (nikul-serpentcs) [reply .github/workflows/test.yml]: Done, @MiquelRForgeFlow Brother :smile:

**Comment by nikul-serpentcs** [comment]
Thanks, @MiquelRForgeFlow, Now your turn

**Comment by MiquelRForgeFlow** [comment]
all green 🍏 🥳 ready to merge!

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis

**Comment by rousseldenis** [comment]
> @nikul-serpentcs please, edit https://github.com/OCA/sale-workflow/blob/13.0/.github/workflows/test.yml#L37 and add case specific for sale_sourced_by_line, and exclude it in the generic case.

@nikul-serpentcs @MiquelRForgeFlow FYI, `copier update` command will ask you to add rebel module and will do the job

**Pull request merged**